@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/event/fn_eventInit.sqf

```diff
@@ -160,7 +160,7 @@ _heli setVariable ["fza_ah64_fcrcscope", false];
 _heli setVariable ["fza_ah64_ihadss_pnvs_cam", false];
 _heli setVariable ["fza_ah64_monocleinbox", true];
 _heli setVariable ["fza_ah64_mpdbrightness", 1];
-_heli setVariable ["fza_ah64_rangesetting", 5000]; //1km
+_heli setVariable ["fza_ah64_scaleRange", 25000]; //25km
```

Comment:
If we are changing this functionality to be on a different variable anyway - it should be in `PersistState`

Reply:
Done